example/imageNet-ft/dataset.yaml

```diff
@@ -0,0 +1,5 @@
+name: image-net-for-hotdog
```

Comment:
only keep dataset_build.py to build dataset? remove `dataset.yaml` and `dataset.py`